ice40/utils/fasm_icebox/fasm_icebox_utils.py

```diff
@@ -0,0 +1,469 @@
+"""Convert the icestorm database to a prjxray type DB
```

Comment:
This file needs to be formatted I think?

Reply:
Done.